app/src/components/Modals/add-activity-modal.tsx

```diff
@@ -49,15 +49,15 @@ import type {
 import { groupBy, resolve } from "@/util/helpers";
 import { ActivityDataScope } from "@/features/city/subsectorSlice";
 
-interface AddUserModalProps {
+interface ActivityDataModalProps {
   isOpen: boolean;
   onClose: () => void;
   t: TFunction;
   userInfo: UserAttributes | null;
   defaultCityId?: string;
   setHasActivityData: Function;
   hasActivityData: boolean;
-  formStruct: ActivityDataScope;
+  formStruct: any;
```

Comment:
Let's add a proper type definition imported from the type inferred from the  zod schema definition here? Or make a new type definition in a `types.ts` file in utils...